tests/read_video.py

```diff
@@ -31,17 +46,28 @@ def run(self):
                 cv2.imshow(display_name, frame)
                 x, y, w, h = cv2.selectROI(display_name, frame, fromCenter=False)
                 self.tracker.init_bounding_box(frame, [x, y, w, h])
+
             if not ret:
                 break
             image, data = self.tracker.run_frame(frame)
             min_x , min_y , w , h, _, _ = data
             image = cv2.rectangle(image, (min_x, min_y), (min_x + w, min_y + h), (0, 255, 0), 5)
```

Comment:
I suggest using name-tuple for colours if used at least twice for better readability (RGB/BGR value). 
Then I suggest extracting the tuple colour to a variable with a meaningful name (e.g., green).